src/lammpsio/topology.py

```diff
@@ -62,7 +62,7 @@ def has_id(self):
         Returns
         -------
         bool
-            True if connection IDs have been initialized.
```

Comment:
Need some comment in Topology that this is a base_class and should not be used directly.